pkg/session_recording/recordings.jsx

```diff
@@ -349,16 +350,22 @@
             cockpit.location.go([recording.id], cockpit.location.options);
         }
 
+        createUrl(recording) {
+            let url = cockpit.location.encode([recording.id], cockpit.location.options);
+            return window.location.origin + "/session_recording#" + url;
```

Comment:
I'm a bit uncomfortable with us hardcoding page location and URL structure here. Couldn't the Cockpit library provide us with the full URL somehow? Could you perhaps ask on #cockpit?

Reply:
Sure, I'll check this, for now I've only this way.

Reply:
One more thing is in cockpit.js window.location is used anyway and some methods are public ( I've also tried to use Location from cockpit.js ) - I'll look more on this and ask, maybe there is a better way.